test/inductor/test_torchinductor_opinfo.py

```diff
@@ -352,8 +352,6 @@ def format_op(op):
     # rrelu not supported on XPU now
     "nn.functional.rrelu": {f16, f32, f64},
     # not implemented for 'Half'
-    "nn.functional.multilabel_margin_loss": {f16},
-    "nn.functional.multi_margin_loss": {f16},
```

Comment:
We remove them from expected failure as we support f16 data type on these ops in torch-xpu-ops.